go/src/koding/klient/client/client_test.go

```diff
@@ -88,13 +143,14 @@ func TestSubscribe(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
+	wait(t, doneC, time.Second)
 
-	_, ok := ps.Subscriptions["test"]
+	subs, ok := getCopy(ps, "test")
```

Comment:
you can remove second bool return value here, return nil map if it's not found and test for len(subs) as you do at L187